change/@microsoft-teams-js-5db60450-58a1-423f-8a07-28fb0d4b515b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "change setcurrentframe to allow all framecontexts",
```

Comment:
As per the Contribution guidelines, can you please change the comment to (emphasis on past tense and use of backticks around types/enums/etc.):

"Changed \`setCurrentFrame\` to allow all \`FrameContexts\`."